src/CountryPicker.tsx

```diff
@@ -45,8 +54,17 @@ export const CountryPicker = forwardRef<CountryPickerRef, CountryPickerProps>(
     const [searchQuery, setSearchQuery] = useState('');
     const debouncedSearchQuery = useDebouncedValue(searchQuery, 300);
 
+    const searchbarRef = useRef<RNPaperTextInputRef>(null);
+
+    const openModal = () => {
+      setVisible(true);
+      setTimeout(() => {
```

Comment:
why use **setTimeout** instead of **useEffect** for this action?

Reply:
Thank you for the review. Unfortunately `useEffect` or `useLayoutEffect` does not work. I assume this is due to the modal opening animation introducing a small delay. 

If I do a `Promise.resolve().then(() => ...)` then the input is focused, but the keyboard doesn't open which is a bit of a confusing UX. I'm not sure if we can avoid the timeout here. 

Cheers.